samples/bluetooth/direction_finding_central/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ /* Enable pin forwarding to network code. The selected pins will be used by
```

Comment:
I'm not familiar with the term `network code` - Can you elaborate?

Reply:
Reading further below: Did you mean `network core`?

Reply:
Applies multiple places

Reply:
And in the commit message :)

Reply:
ah, thanks for spotting that. It should be `network core` everywhere. I assume that there is no need to elaborate about the topic. In case there is such need write it here please.

Comments and commit corrected.